scanner/src/main/kotlin/scanners/ScanCode.kt

```diff
@@ -89,6 +93,107 @@ class ScanCode(name: String, config: ScannerConfiguration) : LocalScanner(name,
         override fun create(config: ScannerConfiguration) = ScanCode(scannerName, config)
     }
 
+    /**
+     * The ScanCode JSON result file format.
+     */
+    data class Result(
+        // This old header format is used by Scancode version 2.x.
+        @JsonProperty("scancode_notice")
+        val notice: String?,
+        @JsonProperty("scancode_version")
+        val version: Semver?,
+        @JsonProperty("scancode_options")
+        val options: Map<String, Any>?,
+        val scanStart: String?, // Not yet available in version 2.2.1.
+        val filesCount: Int?,
```

Comment:
Would it make sense to make this field private to prevent accidentally using it instead of `totalFilesCount`?